paperqa/types.py

```diff
@@ -248,6 +248,10 @@ def could_not_answer(self) -> bool:
         return "cannot answer" in self.answer.lower()
 
 
+# for backwards compatibility
+Answer = PQASession
```

Comment:
I am thinking you don't feel like doing this, but should we start a deprecation cycle for `Answer`?